doc/development/plugins.rst

```diff
@@ -195,6 +204,164 @@ or can include in-built transforms such as:
 * :func:`pennylane.devices.preprocess.validate_adjoint_trainable_params`
 * :func:`pennylane.devices.preprocess.no_sampling`
 
+PennyLane also provides a default implementation of :meth:`~pennylane.devices.Device.preprocess`
+which should be sufficient for most plugin devices. To take advantage of this default implementation,
+you need to define a configuration file for your device as described below.
```

Comment:
I'm not sure if this is the right way to phrase this.  It only has a default implementation if a proper toml file is provided.  It doesn't provided a good default preprocessing without it.

```suggestion
Instead of writing a customized :meth:`~pennylane.devices.Device.preprocess` method, plugin developers an instead provide a toml file configuration, and a default preprocessing will be populated from that file.
```

Or something like that.

Reply:
How about now?